modules/editor/message/MessageEditor.tsx

```diff
@@ -19,15 +22,55 @@ const DataEditorModal = dynamic<DataEditorModalProps>(async () =>
   import("../data/DataEditorModal").then(module => module.DataEditorModal),
 )
 
-const ErrorWrapper = styled.div`
-  margin: 8px 0 0;
+const Message = styled(Markdown)`
+  margin-top: -8px;
+  font-size: 15px;
+`
+
+const EmbedInputWrapper = styled.div`
+  padding-bottom: 16px;
 `
 
 export type MessageEditorProps = {
   message: MessageLike
   form: MessageItemFormState
 }
 
+export function EmbedGroup(props: MessageEditorProps) {
```

Comment:
Don't put 2 components in the same file, this component looks simple enough to just embed in the main component itself.